hedera-node/hedera-file-service-impl/src/main/resources/genesis/simpleFeesSchedules.json

```diff
@@ -154,16 +154,18 @@
       "schedule": [
         {
           "name": "TokenCreate",
-          "baseFee": 25,
+          "baseFee": 10000000000,
           "extras": [
           ]
         },
         {
           "name": "TokenMint",
-          "baseFee": 33,
+          "baseFee": 9000000,
```

Comment:
This is for TokenMintWithFungibleTokens right ? And we will need to add `TokenMintWithFungibleTokens`